addons/game.controller.ps/layout.xml

```diff
@@ -7,19 +7,24 @@
 		<button name="triangle" type="digital" label="30004"/>
 		<button name="start" type="digital" label="30005"/>
 		<button name="select" type="digital" label="30006"/>
-		<button name="up" type="digital" label="30007" direction="up"/>
-		<button name="down" type="digital" label="30008" direction="down"/>
-		<button name="right" type="digital" label="30009" direction="right"/>
-		<button name="left" type="digital" label="30010" direction="left"/>
-		<button name="l3" type="digital" label="30015"/>
-		<button name="r3" type="digital" label="30016"/>
+		<button name="guide" type="digital" label="30007"
```

Comment:
Doesn't look like it, https://github.com/libretro/beetle-psx-libretro/blob/master/libretro.cpp#L2757

I can remove it